citations/citations-servlet/servlet/src/webapp/vm/servlet.vm

```diff
@@ -20,7 +20,7 @@
 		<title>$tlang.getString( "window.gscholar" )</title>
 		<link href="/library/skin/tool_base.css" type="text/css" rel="stylesheet" media="all" />
 		<link href="/library/skin/default/tool.css" type="text/css" rel="stylesheet" media="all" />
-		<script type="text/javascript" src="/library/js/jquery/1.7.1/jquery-1.7.1.min.js"></script>
+		<script type="text/javascript" src="webjars/jquery/1.11.3/jquery.min.js"></script>
```

Comment:
Yes you're right (and I don't think servlet.vm and reorder.vm are used anymore).